src/libs/vmisc/projectversion.cpp

```diff
@@ -63,18 +63,16 @@
 #include <QSysInfo>
 #include <QtGlobal>
 
+// Start: Do not edit here, use scripts/version.sh to update
 extern const int MAJOR_VERSION = 0;
 extern const int MINOR_VERSION = 6;
```

Comment:
Are Major Version and Minor Version static? Can you comment where in the code these values are updated?